### ISSUE TITLE
Heroku error on latest build

### ISSUE THREAD

**Issue by cesy** [Growstuff/growstuff#1389]
bundle exec script/heroku_maintenance.rb on
Turning 1 maintenance mode on app growstuff-staging
bundler: failed to load command: script/heroku_maintenance.rb (script/heroku_maintenance.rb)
Heroku::API::Errors::ErrorWithResponse: Expected(200) <=> Actual(400 Bad Request)
body: "{\"id\":\"missing_version\",\"error\":\"Please specify a version along with Heroku's API MIME type. For example, `Accept: application/vnd.heroku+json; version=3`.\\n\"}"

**Comment by cesy** [comment]
Looks like this means all our deployments have been failing for about 2 months. Need to figure out what version we're using and add it.

**Comment by Br3nda** [comment]
```
0.30s$ bundle exec script/heroku_maintenance.rb on
bundler: failed to load command: script/heroku_maintenance.rb (script/heroku_maintenance.rb)
LoadError: cannot load such file -- heroku-api
  script/heroku_maintenance.rb:3:in `require'
  script/heroku_maintenance.rb:3:in `<top (required)>'
The command "bundle exec script/heroku_maintenance.rb on" failed and exited with 1 during .
Your build has been stopped.
```

**Comment by pozorvlak** [comment]
Can we just replace `require 'heroku-api'` with `require 'platform-api'` in `script/heroku_maintenance.rb`?

I wondered if we could delete that script entirely and replace it with something built in to Travis, but [it doesn't look like it](https://github.com/travis-ci/dpl/issues/105).

**Comment by pozorvlak** [comment]
Fixed in #1402.
